### ISSUE TITLE
AuthResult type should be exported in amplify_auth_cognito_dart

### ISSUE THREAD

**Issue by pitriq** [aws-amplify/amplify-flutter#2623]
Hey! I noticed [this PR](https://github.com/aws-amplify/amplify-flutter/pull/2585) was merged in the last preview release. 

It introduces `AuthResult` as the type of some fields in `CognitoAuthSession`, but it is not exported, so we can't access it in our app. I may be wrong but I think this would be desirable, as we need it to mock the return value of `fetchAuthSession()` in our tests.

What do you guys think? Cheers.

**Comment by Jordan-Nelson** [comment]
@pitriq - Thanks for opening the issue. I just merged a PR to export this. It will be included in our next developer-preview release. Until then, I believe you can import this from the `/src` dir in tests, although it may result in analysis issues depending on where you import it and what lint rules you have enabled.


```dart
import 'package:amplify_auth_cognito_dart/src/model/auth_result.dart';
```

**Comment by pitriq** [comment]
Great! Thanks for the PR & the tip. Feel free to close this.

**Comment by dnys1** [comment]
This has been released in amplify_auth_cognito_dart v0.6.0. Please let us know if you continue to face issues.
